### PR TITLE
Clarify copy-reporter addon description

### PR DESCRIPTION
**Pull request by MaterArc**
Clarified the description.

### CODE REVIEW

**Comment by WorldLanguages** [comment]
> Adds buttons to copy the value of reporter balloons and monitors by right clicking the variable readout on the project.

I'm not sure if it's good enough. It should be clear that these are 2 separate features - copying the value of reporter balloons, and separately, right clicking variables on the **stage** (clearer than "project") and clicking the new "copy value" option.

File: addons/copy-reporter/addon.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "Copy reporter values",
-  "description": "Adds buttons to copy the value of reporter balloons and monitors.",
+  "description": "Adds buttons to copy the value of reporter balloons and monitors by right clicking the variable readout on the project.",
```
Comment (Samq64) [inline addons/copy-reporter/addon.json:3]:
```suggestion
  "description": "Adds copy value buttons to block reporter balloons and variable monitor context menus.",
```

Comment (WorldLanguages) [reply addons/copy-reporter/addon.json:3]: We often use "right click context menus" for clarity.

Comment (MaterArc) [reply addons/copy-reporter/addon.json:3]: Ok! :)

File: addons/copy-reporter/addon.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "Copy reporter values",
-  "description": "Adds buttons to copy the value of reporter balloons and monitors.",
+  "description": "Adds copy value buttons to block reporter balloons and variable monitor context menus.",
```
Comment (DNin01) [inline addons/copy-reporter/addon.json:3]: I have another idea:
```suggestion
  "description": "Adds the option to copy values displayed by variable monitors (from the right click context menu) and block reporter balloons.",
```

**Comment by WorldLanguages** [comment]
Merging DNin's version, which has 2 upvotes

**Pull request merged**